recipes-bsp/firmware-imx/firmware-ele-imx_0.1.3.bb

```diff
@@ -19,9 +18,9 @@ do_compile[noexec] = "1"
 do_install() {
    install -d ${D}${nonarch_base_libdir}/firmware/imx/ele
    install -m 0644 ${S}/${SECO_FIRMWARE_NAME} ${D}${nonarch_base_libdir}/firmware/imx/ele
-   if [ -e ${S}/${SECOEXT_FIRMWARE_NAME} ]; then
-       install -m 0644 ${S}/${SECOEXT_FIRMWARE_NAME} ${D}${nonarch_base_libdir}/firmware/imx/ele
-   fi
+    if [ -e ${S}/${SECOEXT_FIRMWARE_NAME} ]; then
+        install -m 0644 ${S}/${SECOEXT_FIRMWARE_NAME} ${D}${nonarch_base_libdir}/firmware/imx/ele
+    fi
```

Comment:
This will break the alignment, not? I think it is correct and this is changing to add a unneeded space.

Reply:
This is the original source/style from NXP's meta-imx: https://github.com/nxp-imx/meta-imx/blob/scarthgap-6.6.36-2.1.0/meta-imx-bsp/recipes-bsp/firmware-imx/firmware-ele-imx_0.1.3.bb#L21

Reply:
So it seems they need to fix their code. @thochstein Is it possible for you to take a look on that?

Reply:
This code parts should works, it is just an ugly style, but firstly it need to be fixed in [meta-imx](https://github.com/nxp-imx/meta-imx).

Reply:
We can just not import the wrong style when updating the recipe.

Reply:
What's being imported is 'right style' to me :-) I will fix our recipe to 4-space indentation.

Reply:
Style fix done!